### PR TITLE
remove lockfiles

### PR DESCRIPTION
**Pull request by etcart**
this pr assumes that lockfiles 
- cumulus/.terraform.lock.hcl
- data-persistence/.terraform.lock.hcl
- tf/.terraform.lock.hcl

were added in error
this is because they are not present in previous releases, nor a part of any release notes

and this pr removes them

### CODE REVIEW

**Review by gjclark — DISMISSED** [review]

**Comment by mattp0** [comment]
Hello @etcart 
Can you describe the error you are having in more detail? The lock files are a way to ensure a provider version is pinned. 
https://developer.hashicorp.com/terraform/language/files/dependency-lock 
I do not believe they were added erroneously.  Generally the code that is added has been tested.

**Comment by mattp0** [comment]
Here is a bit more context on the lockfiles from a good stack overflow post https://stackoverflow.com/questions/67963719/should-terraform-lock-hcl-be-included-in-the-gitignore-file

**Comment by mattp0** [comment]
Are you seeing a checksum not matching error?

**Comment by etcart** [comment]
the error we are getting is
```
Error: Resource instance managed by newer provider version
The current state of aws_cloudwatch_event_rule.background_job_queue_watcher
was created by a newer provider version than is currently selected. Upgrade
the aws provider to work with this state.
```
this occuring as we are updating from v17.0.0.2, which does not have lockfiles coming from cirrus-core, to v18.0.0.0, which has lockfiles coming from cirrus-core, but not cirrus-daac.
because we have been on an unlocked version of cirrus, we have deployed with aws provider version 5.31, which now isn't workable when we have to downgrade to aws provider version 5.20

**Comment by mattp0** [comment]
> the error we are getting is
> 
> ```
> Error: Resource instance managed by newer provider version
> The current state of aws_cloudwatch_event_rule.background_job_queue_watcher
> was created by a newer provider version than is currently selected. Upgrade
> the aws provider to work with this state.
> ```
> 
> this occuring as we are updating from v17.0.0.2, which does not have lockfiles coming from cirrus-core, to v18.0.0.0, which has lockfiles coming from cirrus-core, but not cirrus-daac. because we have been on an unlocked version of cirrus, we have deployed with aws provider version 5.31, which now isn't workable when we have to downgrade to aws provider version 5.20

OK. I think this covers and edge case that maybe we werent expecting. Have you been able to confirm that this branch now deploys? In your CI you should be able to point to this feature branch instead of the release ref.

**Comment by etcart** [comment]
precisely, by pointing to this branch we're working just fine

**Review by mattp0 — APPROVED** [review]

**Comment by etcart** [comment]
if the intention is to lock to aws provider 5.20, shouldn't it be set in the provider.tf file? and also be set in the relevant release of CIRRUS-DAAC as well as CIRRUS-core?

**Review by reweeden — COMMENTED** [review]
You should add `*.lock.hcl` to the .gitignore to prevent the files from being added accidentally.

**Review by mckadesorensen — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@ workflows
 scripts/**/*.log
 .container_bash_history
 .venv
+*.lock.hcl
```
Comment (reweeden) [inline .gitignore:24]: Missing newline at end of file

**Pull request merged**